NDTensors/src/lib/GradedAxes/src/gradedunitrangedual.jl

```diff
@@ -0,0 +1,148 @@
+struct GradedUnitRangeDual{
+  T<:LabelledInteger,NondualUnitRange<:AbstractGradedUnitRange{T}
+} <: AbstractGradedUnitRange{T,Vector{T}}
```

Comment:
```suggestion
struct GradedUnitRangeDual{
  T,NondualUnitRange<:AbstractGradedUnitRange{T,CS}
} <: AbstractGradedUnitRange{T,CS}
```
to make it a bit more general.